### DISCUSSION TITLE
Speed of execution compared to sklearns GP library

### DISCUSSION THREAD

**Discussion by azinoma** [GPflow/GPflow#1628] [Q&A]
When I try fitting simple examples with 20 - 300 points using GPFlow, I noticed that the execution speed for the prediction is significantly (more than 20x) lower than the execution speed of the prediction when using sklearns GP library.
I am wondering whether this issue is known or if I am maybe messing something up somewhere?

**Comment by st--** [comment]
Hi @azinoma, this had been a recurring question (#1554, #1189, #1030, #1003). We worked on addressing this issue with https://github.com/GPflow/GPflow/issues/1599. It is part of GPflow as of version 2.2. Please check if this resolves your issue:)
